src/GeneralStateTestsFiller/stExample/03_tloadAfterStoreIs0Filler.yml

```diff
@@ -0,0 +1,63 @@
+# 03
+# Loading any other slot after storing to a slot returns 0: TSTORE(x, y), TLOAD(x + n) where n > 0 returns 0
+#
+# Expect storage slot 0 to have value 0
+# Ex 1) stores 30 at slot 0, loads from slot 1, expects storage slot 0 to have 0
+
+03_tloadAfterStoreIs0:
+  _info:
+    comment: Loading any other slot after storing to a slot returns 0.
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x10000000000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+  pre:
+    000000000000000000000000000000000000000A:
+      balance: 1000000000000000000
+      nonce: 0
+      code: |
+        :yul {
+          verbatim_2i_0o(hex"b4", 0, 30)
+          let val := verbatim_1i_1o(hex"b3", 1)
+          sstore(0, val)
```

Comment:
again you expect 0 value as result, set initial storage at key 0x00 to dummy 0xffff  this is just a percaution in case transaction oogs.